Tests/AblyChatTests/Mocks/MockRealtimePresence.swift

```diff
@@ -63,3 +63,60 @@ extension ARTRealtimePresenceQuery {
         "clientId=\(clientId!)"
     }
 }
+
+final class MockRealtimeAnnotations: InternalRealtimeAnnotationsProtocol {
+    let annotationJSONToEmitOnSubscribe: [String: JSONValue]?
```

Comment:
Why can't this just be an `ARTAnnotation` object?

Reply:
done